airflow/providers/google/cloud/operators/bigquery.py

```diff
@@ -1001,7 +1063,7 @@ def execute_complete(self, context: Context, event: dict[str, Any]) -> Any:
         return event["records"]
 
 
-class BigQueryExecuteQueryOperator(GoogleCloudBaseOperator):
+class BigQueryExecuteQueryOperator(GoogleCloudBaseOperator, _BigQueryOpenLineageMixin):
```

Comment:
I'm -1 for adding new features for deprecated classes.

Reply:
@eladkal removed this from `BigQueryExecuteQueryOperator` and associated tests.